org.supertuxproject.SuperTux.json

```diff
@@ -108,9 +113,11 @@
                     "path": "supertux-build-fixes.patch"
                 },
                 {
-                    "//": "Remember to delete this patch when the next SuperTux version gets released, as the appdata fix has also been merged into SuperTux's source code: https://github.com/SuperTux/supertux/pull/2668",
-                    "type": "patch",
-                    "path": "supertux-appdata.patch"
+
+                    "type": "file",
+                    "url": "https://raw.githubusercontent.com/SuperTux/supertux/12cf6c012e2bc1b91ef12fe56df1b169b2a5fa47/supertux2.appdata.xml",
+                    "sha256": "f6e87e02dd11037fdade5e65ebb8c956140129e71dd4d00ec9435fc1dab1f096",
+                    "dest-filename": "updated-appdata.xml"
```

Comment:
this completely ignore line 9. which is why things aren't working the way want.

what's wrong with the patch?

Reply:
The updated appdata file linked here has some other changes made to it, besides the one made by the patch, that help fix [the metainfo validation errors described here](https://github.com/flathub/org.supertuxproject.SuperTux/issues/20#issuecomment-1816947323).

Reply:
then the patch should have been updated instead

Reply:
> then the patch should have been updated instead

Ah. Didn't even think of that :man_facepalming: